extensions/cornerstone/src/utils/measurementServiceMappings/NLFreehandRoi.js

```diff
@@ -0,0 +1,162 @@
+import cornerstone from 'cornerstone-core';
```

Comment:
Do you mind explaining how this is different from https://github.com/cornerstonejs/cornerstoneTools/blob/master/src/tools/annotation/FreehandRoiTool.js and why we need a separate tool file for this one instead of EllipticalRoi for example? Is it because of the measurement sidebar mapping?